diff --git a/node_modules/@radix-ui/react-context-menu/.DS_Store b/node_modules/@radix-ui/react-context-menu/.DS_Store
new file mode 100644
index 0000000..c88a062
Binary files /dev/null and b/node_modules/@radix-ui/react-context-menu/.DS_Store differ
diff --git a/node_modules/@radix-ui/react-context-menu/dist/index.module.js b/node_modules/@radix-ui/react-context-menu/dist/index.module.js
index b3a6715..7d2ade6 100644
--- a/node_modules/@radix-ui/react-context-menu/dist/index.module.js
+++ b/node_modules/@radix-ui/react-context-menu/dist/index.module.js
@@ -107,12 +107,13 @@ const $1b0217ee4a91d156$export$be6fc7b1d5b0beb9 = /*#__PURE__*/ $3TiJz$forwardRe
             handleOpen(event);
             event.preventDefault();
         }),
-        onPointerDown: disabled ? props.onPointerDown : $3TiJz$composeEventHandlers(props.onPointerDown, $1b0217ee4a91d156$var$whenTouchOrPen((event)=>{
-            // clear the long press here in case there's multiple touch points
+        onPointerDown: disabled ? props.onContextMenu : $3TiJz$composeEventHandlers(props.onContextMenu, (event)=>{
+            // clearing the long press here because some platforms already support
+            // long press to trigger a `contextmenu` event
             clearLongPress();
-            longPressTimerRef.current = window.setTimeout(()=>handleOpen(event)
-            , 700);
-        })),
+            handleOpen(event);
+            event.preventDefault();
+        }),
         onPointerMove: disabled ? props.onPointerMove : $3TiJz$composeEventHandlers(props.onPointerMove, $1b0217ee4a91d156$var$whenTouchOrPen(clearLongPress)),
         onPointerCancel: disabled ? props.onPointerCancel : $3TiJz$composeEventHandlers(props.onPointerCancel, $1b0217ee4a91d156$var$whenTouchOrPen(clearLongPress)),
         onPointerUp: disabled ? props.onPointerUp : $3TiJz$composeEventHandlers(props.onPointerUp, $1b0217ee4a91d156$var$whenTouchOrPen(clearLongPress))
