diff --git a/node_modules/@sentry/core/esm/integrations/functiontostring.js b/node_modules/@sentry/core/esm/integrations/functiontostring.js
index ef0ca57..d7ea680 100644
--- a/node_modules/@sentry/core/esm/integrations/functiontostring.js
+++ b/node_modules/@sentry/core/esm/integrations/functiontostring.js
@@ -19,13 +19,13 @@ class FunctionToString  {constructor() { FunctionToString.prototype.__init.call(
    */
    setupOnce() {
     // eslint-disable-next-line @typescript-eslint/unbound-method
-    originalFunctionToString = Function.prototype.toString;
+    // originalFunctionToString = Function.prototype.toString;
 
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    Function.prototype.toString = function ( ...args) {
-      const context = getOriginalFunction(this) || this;
-      return originalFunctionToString.apply(context, args);
-    };
+    // Function.prototype.toString = function ( ...args) {
+    //   const context = getOriginalFunction(this) || this;
+    //   return originalFunctionToString.apply(context, args);
+    // };
   }
 } FunctionToString.__initStatic();
 
