diff --git a/node_modules/@wagmi/core/dist/connectors/coinbaseWallet.d.ts b/node_modules/@wagmi/core/dist/connectors/coinbaseWallet.d.ts
deleted file mode 100644
index 4e3d9c7..0000000
--- a/node_modules/@wagmi/core/dist/connectors/coinbaseWallet.d.ts
+++ /dev/null
@@ -1 +0,0 @@
-export { CoinbaseWalletConnector } from '@wagmi/connectors/coinbaseWallet';
diff --git a/node_modules/@wagmi/core/dist/connectors/coinbaseWallet.js b/node_modules/@wagmi/core/dist/connectors/coinbaseWallet.js
deleted file mode 100644
index a059bbb..0000000
--- a/node_modules/@wagmi/core/dist/connectors/coinbaseWallet.js
+++ /dev/null
@@ -1,7 +0,0 @@
-import "../chunk-MQXBDTVK.js";
-
-// src/connectors/coinbaseWallet.ts
-import { CoinbaseWalletConnector } from "@wagmi/connectors/coinbaseWallet";
-export {
-  CoinbaseWalletConnector
-};
diff --git a/node_modules/@wagmi/core/dist/connectors/index.d.ts b/node_modules/@wagmi/core/dist/connectors/index.d.ts
deleted file mode 100644
index 3572432..0000000
--- a/node_modules/@wagmi/core/dist/connectors/index.d.ts
+++ /dev/null
@@ -1,2 +0,0 @@
-export { Connector, ConnectorData, ConnectorEvents } from '@wagmi/connectors';
-export { InjectedConnector, InjectedConnectorOptions } from '@wagmi/connectors/injected';
diff --git a/node_modules/@wagmi/core/dist/connectors/index.js b/node_modules/@wagmi/core/dist/connectors/index.js
deleted file mode 100644
index fd6a641..0000000
--- a/node_modules/@wagmi/core/dist/connectors/index.js
+++ /dev/null
@@ -1,9 +0,0 @@
-import {
-  Connector,
-  InjectedConnector
-} from "../chunk-BVC4KGLQ.js";
-import "../chunk-MQXBDTVK.js";
-export {
-  Connector,
-  InjectedConnector
-};
diff --git a/node_modules/@wagmi/core/dist/connectors/injected.d.ts b/node_modules/@wagmi/core/dist/connectors/injected.d.ts
deleted file mode 100644
index 0fbfbbb..0000000
--- a/node_modules/@wagmi/core/dist/connectors/injected.d.ts
+++ /dev/null
@@ -1 +0,0 @@
-export { InjectedConnector, InjectedConnectorOptions } from '@wagmi/connectors/injected';
diff --git a/node_modules/@wagmi/core/dist/connectors/injected.js b/node_modules/@wagmi/core/dist/connectors/injected.js
deleted file mode 100644
index a6ae76a..0000000
--- a/node_modules/@wagmi/core/dist/connectors/injected.js
+++ /dev/null
@@ -1,7 +0,0 @@
-import {
-  InjectedConnector
-} from "../chunk-EQOEZP46.js";
-import "../chunk-MQXBDTVK.js";
-export {
-  InjectedConnector
-};
diff --git a/node_modules/@wagmi/core/dist/connectors/ledger.d.ts b/node_modules/@wagmi/core/dist/connectors/ledger.d.ts
deleted file mode 100644
index 3cc3715..0000000
--- a/node_modules/@wagmi/core/dist/connectors/ledger.d.ts
+++ /dev/null
@@ -1 +0,0 @@
-export { LedgerConnector } from '@wagmi/connectors/ledger';
diff --git a/node_modules/@wagmi/core/dist/connectors/ledger.js b/node_modules/@wagmi/core/dist/connectors/ledger.js
deleted file mode 100644
index e980eaf..0000000
--- a/node_modules/@wagmi/core/dist/connectors/ledger.js
+++ /dev/null
@@ -1,7 +0,0 @@
-import "../chunk-MQXBDTVK.js";
-
-// src/connectors/ledger.ts
-import { LedgerConnector } from "@wagmi/connectors/ledger";
-export {
-  LedgerConnector
-};
diff --git a/node_modules/@wagmi/core/dist/connectors/metaMask.d.ts b/node_modules/@wagmi/core/dist/connectors/metaMask.d.ts
deleted file mode 100644
index 568143f..0000000
--- a/node_modules/@wagmi/core/dist/connectors/metaMask.d.ts
+++ /dev/null
@@ -1 +0,0 @@
-export { MetaMaskConnector, MetaMaskConnectorOptions } from '@wagmi/connectors/metaMask';
diff --git a/node_modules/@wagmi/core/dist/connectors/metaMask.js b/node_modules/@wagmi/core/dist/connectors/metaMask.js
deleted file mode 100644
index 18d2a06..0000000
--- a/node_modules/@wagmi/core/dist/connectors/metaMask.js
+++ /dev/null
@@ -1,7 +0,0 @@
-import "../chunk-MQXBDTVK.js";
-
-// src/connectors/metaMask.ts
-import { MetaMaskConnector } from "@wagmi/connectors/metaMask";
-export {
-  MetaMaskConnector
-};
diff --git a/node_modules/@wagmi/core/dist/connectors/mock.d.ts b/node_modules/@wagmi/core/dist/connectors/mock.d.ts
deleted file mode 100644
index 4e7f86a..0000000
--- a/node_modules/@wagmi/core/dist/connectors/mock.d.ts
+++ /dev/null
@@ -1 +0,0 @@
-export { MockConnector, MockProvider, MockProviderOptions } from '@wagmi/connectors/mock';
diff --git a/node_modules/@wagmi/core/dist/connectors/mock.js b/node_modules/@wagmi/core/dist/connectors/mock.js
deleted file mode 100644
index d70b138..0000000
--- a/node_modules/@wagmi/core/dist/connectors/mock.js
+++ /dev/null
@@ -1,9 +0,0 @@
-import {
-  MockConnector,
-  MockProvider
-} from "../chunk-KFW652VN.js";
-import "../chunk-MQXBDTVK.js";
-export {
-  MockConnector,
-  MockProvider
-};
diff --git a/node_modules/@wagmi/core/dist/connectors/safe.d.ts b/node_modules/@wagmi/core/dist/connectors/safe.d.ts
deleted file mode 100644
index d0292c9..0000000
--- a/node_modules/@wagmi/core/dist/connectors/safe.d.ts
+++ /dev/null
@@ -1 +0,0 @@
-export { SafeConnector, SafeConnectorOptions, SafeConnectorProvider } from '@wagmi/connectors/safe';
diff --git a/node_modules/@wagmi/core/dist/connectors/safe.js b/node_modules/@wagmi/core/dist/connectors/safe.js
deleted file mode 100644
index ae78fa7..0000000
--- a/node_modules/@wagmi/core/dist/connectors/safe.js
+++ /dev/null
@@ -1,7 +0,0 @@
-import "../chunk-MQXBDTVK.js";
-
-// src/connectors/safe.ts
-import { SafeConnector } from "@wagmi/connectors/safe";
-export {
-  SafeConnector
-};
diff --git a/node_modules/@wagmi/core/dist/connectors/walletConnect.d.ts b/node_modules/@wagmi/core/dist/connectors/walletConnect.d.ts
deleted file mode 100644
index fee8063..0000000
--- a/node_modules/@wagmi/core/dist/connectors/walletConnect.d.ts
+++ /dev/null
@@ -1 +0,0 @@
-export { WalletConnectConnector } from '@wagmi/connectors/walletConnect';
diff --git a/node_modules/@wagmi/core/dist/connectors/walletConnect.js b/node_modules/@wagmi/core/dist/connectors/walletConnect.js
deleted file mode 100644
index 30e7306..0000000
--- a/node_modules/@wagmi/core/dist/connectors/walletConnect.js
+++ /dev/null
@@ -1,7 +0,0 @@
-import "../chunk-MQXBDTVK.js";
-
-// src/connectors/walletConnect.ts
-import { WalletConnectConnector } from "@wagmi/connectors/walletConnect";
-export {
-  WalletConnectConnector
-};
diff --git a/node_modules/@wagmi/core/dist/connectors/walletConnectLegacy.d.ts b/node_modules/@wagmi/core/dist/connectors/walletConnectLegacy.d.ts
deleted file mode 100644
index e66064a..0000000
--- a/node_modules/@wagmi/core/dist/connectors/walletConnectLegacy.d.ts
+++ /dev/null
@@ -1 +0,0 @@
-export { WalletConnectLegacyConnector } from '@wagmi/connectors/walletConnectLegacy';
diff --git a/node_modules/@wagmi/core/dist/connectors/walletConnectLegacy.js b/node_modules/@wagmi/core/dist/connectors/walletConnectLegacy.js
deleted file mode 100644
index 0bf0466..0000000
--- a/node_modules/@wagmi/core/dist/connectors/walletConnectLegacy.js
+++ /dev/null
@@ -1,7 +0,0 @@
-import "../chunk-MQXBDTVK.js";
-
-// src/connectors/walletConnectLegacy.ts
-import { WalletConnectLegacyConnector } from "@wagmi/connectors/walletConnectLegacy";
-export {
-  WalletConnectLegacyConnector
-};
