diff --git a/node_modules/@wagmi/core/dist/declarations/src/types/index.d.ts b/node_modules/@wagmi/core/dist/declarations/src/types/index.d.ts
index 873efa1..59a9194 100644
--- a/node_modules/@wagmi/core/dist/declarations/src/types/index.d.ts
+++ b/node_modules/@wagmi/core/dist/declarations/src/types/index.d.ts
@@ -187,9 +187,9 @@ export interface Ethereum extends InjectedProviders {
         }];
     }): Promise<null>;
 }
-declare global {
-    interface Window {
-        ethereum?: Ethereum;
-    }
-}
+// declare global {
+//     interface Window {
+//         ethereum?: Ethereum;
+//     }
+// }
 export {};
