diff --git a/node_modules/browserify-hmr/inc/index.js b/node_modules/browserify-hmr/inc/index.js
index afda18d..78d2773 100644
--- a/node_modules/browserify-hmr/inc/index.js
+++ b/node_modules/browserify-hmr/inc/index.js
@@ -527,7 +527,13 @@ function main(
           }
         });
       };
-      socket.on('new modules', acceptNewModules);
+      socket.on('new modules', () => {
+        if (!!window.location && typeof window.location.reload === "function") {
+          window.location.reload()
+        } else {
+          chrome.runtime.reload()
+        }
+      });
       return socket;
     };
 
