# These patches are required for LavaMoat Node compat
diff --git a/node_modules/typescript/lib/typescript.js b/node_modules/typescript/lib/typescript.js
index c0f6f3a..94a12b6 100644
--- a/node_modules/typescript/lib/typescript.js
+++ b/node_modules/typescript/lib/typescript.js
@@ -10282,7 +10282,7 @@ var ts;
             continue: 86 /* SyntaxKind.ContinueKeyword */,
             const: 85 /* SyntaxKind.ConstKeyword */
         },
-        _a["" + "constructor"] = 134 /* SyntaxKind.ConstructorKeyword */,
+        // _a["" + "constructor"] = 134 /* SyntaxKind.ConstructorKeyword */,
         _a.debugger = 87 /* SyntaxKind.DebuggerKeyword */,
         _a.declare = 135 /* SyntaxKind.DeclareKeyword */,
         _a.default = 88 /* SyntaxKind.DefaultKeyword */,
@@ -170433,7 +170433,7 @@ var ts;
         // In IE8, Object.defineProperty only works on DOM objects.
         // If we hit this code path, assume `window` exists.
         //@ts-ignore
-        window.globalThis = window;
+        // window.globalThis = window;
     }
 })());
 // #endregion The polyfill ends here.
