diff --git a/node_modules/typescript/lib/typescript.js b/node_modules/typescript/lib/typescript.js
index c2688dd..453f002 100644
--- a/node_modules/typescript/lib/typescript.js
+++ b/node_modules/typescript/lib/typescript.js
@@ -10157,7 +10157,7 @@ var ts;
             continue: 86 /* SyntaxKind.ContinueKeyword */,
             const: 85 /* SyntaxKind.ConstKeyword */
         },
-        _a["" + "constructor"] = 135 /* SyntaxKind.ConstructorKeyword */,
+        // _a["" + "constructor"] = 135 /* SyntaxKind.ConstructorKeyword */,
         _a.debugger = 87 /* SyntaxKind.DebuggerKeyword */,
         _a.declare = 136 /* SyntaxKind.DeclareKeyword */,
         _a.default = 88 /* SyntaxKind.DefaultKeyword */,
@@ -172072,7 +172072,7 @@ var ts;
         // In IE8, Object.defineProperty only works on DOM objects.
         // If we hit this code path, assume `window` exists.
         //@ts-ignore
-        window.globalThis = window;
+        // window.globalThis = window;
     }
 })());
 // #endregion The polyfill ends here.
